Override shell-quote to ^1.9.0.
Remove when upstream Code-OSS updates shell-quote to >= 1.9.0.

@generated
@generator: scripts/patches/apply-override.sh --patch common/finding-override-shell-quote.diff --override 'global:shell-quote=^1.9.0' --override 'remote/package.json@global:shell-quote=^1.9.0'
@override-package: shell-quote@^1.9.0

Index: code-editor-src/package.json
===================================================================
--- code-editor-src.orig/package.json
+++ code-editor-src/package.json
@@ -246,6 +246,7 @@
     "yaserver": "^0.4.0"
   },
   "overrides": {
+    "shell-quote": "^1.9.0",
     "postcss": "^8.4.31",
     "node-gyp-build": "4.8.1",
     "serialize-javascript": "^7.0.3",
Index: code-editor-src/remote/package.json
===================================================================
--- code-editor-src.orig/remote/package.json
+++ code-editor-src/remote/package.json
@@ -48,6 +48,7 @@
     "yazl": "^2.4.3"
   },
   "overrides": {
+    "shell-quote": "^1.9.0",
     "node-gyp-build": "4.8.1",
     "ssh2": {
       "cpu-features": "0.0.0"
